code/modules/jobs/job_types/botanist.dm

```diff
@@ -4,7 +4,7 @@
 	department_head = list("Head of Personnel")
 	department_flag = CIVILIAN
 	faction = "Station"
-	total_positions = 3
+	total_positions = 6
```

Comment:
3 botanists are enough, 6 is a absolute overkill

Reply:
botany has enough trays to make the whole crew farm rice